app/js/controllers/dev.js

```diff
@@ -0,0 +1,21 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import CameraLatestImageView from 'js/views/dev/camera-latest-image';
+
+export default class DevController extends Controller {
```

Comment:
I just discussed with @arcturus and we think that redirecting the app to specific routes when we get a message from a web push notification is something that's going to stay.
We should rename `DevController` to a better name (like `ServiceGetterController` or `GetterController` or anything else).

The route too should reflect that. I was thinking a sub route in service like `http://localhost:8000/#service/service:18db9b78f188-11e5-9ce9-3ca9f47e346c@link.mozilla.org/latest-image`, but I leave it to you as long as the URL structure is logical and makes sense.